### PR TITLE
[chore] Set wb-timing as private and modify pre-publish check

### PR DESCRIPTION
**Pull request by jandrade**
## Internal change
- Sets `wonder-blocks-timing` as private to avoid publishing this package.
- Adds an extra pre-publish check to avoid publishing private packages.

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 62f4633f24e684e8b1ae1de4df7f40f6e2aa6b43

https://deploy-preview-412--wonder-blocks.netlify.com

File: utils/pre-publish-check.js [hunk 1]
```diff
@@ -8,10 +8,21 @@ const {exec} = require("child_process");
...
+            `ERROR: ${name} is missing a "publishConfig": {"access": "${requiredAccessType}"} section.`,
+        );
+        process.exit(1);
+    }
+
+    // also check if is marked as private and there's restricted access defined
+    if (isPrivate && publishConfig.access !== "restricted") {
         console.error(
-            `ERROR: ${name} is missing a "publishConfig": {"access": "public"} section.`,
+            `ERROR: ${name} is marked as private but there is a "publishConfig": {"access": "public"} section already defined. Please change it to "access": "restricted".`,
```
Comment (somewhatabstract) [inline utils/pre-publish-check.js]: These checks are great! Thanks!

Comment (jeresig) [inline utils/pre-publish-check.js]: You could add: "... or remove the `"private": true` to make the package public."

File: utils/pre-publish-check.js [hunk 2]
```diff
@@ -65,9 +76,8 @@ glob(
 
         for (const pkgPath of pkgPaths) {
             const pkgJson = require(path.relative(__dirname, pkgPath));
-            const {publishConfig} = pkgJson;
 
-            checkPublishConfig(publishConfig);
+            checkPublishConfig(pkgJson);
```
Comment (somewhatabstract) [inline utils/pre-publish-check.js]: Nice. I like that these all take the same input now.

**Pull request merged**